var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -61,14 +70,22 @@ class Opencv(Package):
             description='Enables the build of Python extensions')
     variant('java', default=False,
             description='Activates support for Java')
+    variant('openmp', default=False, description='Activates support for OpenMP threads')
+    variant('core', default=False, description='Include opencv_core module into the OpenCV build')
+    variant('highgui', default=False, description='Include opencv_highgui module into the OpenCV build')
+    variant('imgproc', default=False, description='Include opencv_imgproc module into the OpenCV build')
+    variant('jpeg', default=False, description='Include JPEG support')
+    variant('png', default=False, description='Include PNG support')
+    variant('tiff', default=False, description='Include TIFF support')
+    variant('zlib', default=False, description='Build zlib from source')
```

Comment:
You add a lot of variants but you don't add any dependencies. You should be making some of these dependencies optional, correct?